lighthouse-core/lib/log.js

```diff
@@ -113,23 +115,23 @@ class Log {
     Log._logToStdErr(`${prefix}:${level || ''}`, [data.method, snippet]);
   }
 
-  static log(title) {
-    Log.events.issueStatus(title, arguments);
-    return Log._logToStdErr(title, Array.from(arguments).slice(1));
+  static log(title, ...args) {
+    Log.events.issueStatus(title, ...args);
+    return Log._logToStdErr(title, args);
   }
 
-  static warn(title) {
-    Log.events.issueWarning(arguments);
-    return Log._logToStdErr(`${title}:warn`, Array.from(arguments).slice(1));
+  static warn(title, ...args) {
+    Log.events.issueWarning(title, ...args);
+    return Log._logToStdErr(`${title}:warn`, args);
   }
 
-  static error(title) {
-    return Log._logToStdErr(`${title}:error`, Array.from(arguments).slice(1));
+  static error(title, ...args) {
+    return Log._logToStdErr(`${title}:error`, args);
   }
 
-  static verbose(title) {
+  static verbose(title, ...args) {
     Log.events.issueStatus(title);
```

Comment:
> See comment above about arguments, ...args inside the function will be an empty []

I don't think that applies here since it's only doing `Log.events.issueStatus(title)`, so it gets an empty array on master already.

Honestly I have no idea why it's doing `issueStatus` here with no args anyways. It doesn't make much sense.